tests/nested/core/core-fde-state/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: Check the FDE state is correctly initialized on first boot
+
+details: |
+    This test checks checks that the FDE state is correctly
+    initialized on first boot.
+
+systems: [ubuntu-20.04-64, ubuntu-22.04-64, ubuntu-24.04-64]
```

Comment:
shouldn't the filters be of the form ubuntu-core-* ? for tests/core tests?

Reply:
in nested the UC version is implied by host system, so that ubuntu-20.04 is for uc20 testing and so on, see https://github.com/canonical/snapd/blob/8c0a7412b4143522d6afaa8a97bdcd282840be86/tests/lib/tools/tests.nested#L420-L435

I don't think this distinction makes sense anymore, perhaps we should look into using the latest release as host for all UC versions.

Reply:
I was aware, I somehow missed the nested when I looked at that path :/